### DISCUSSION TITLE
Update docker-compose.yml to remove version obsolete warning

### DISCUSSION THREAD

**Discussion by derek-palmer** [AbdBarho/stable-diffusion-webui-docker#691] [General]
Proposal to remove `version:` syntax in docker-compose.yml. If left in, it throws an obsolete warning. 

This reduces unnecessary warnings/chatter and keeps the code up to current standards. See [Version top-level element (obsolete)](https://docs.docker.com/compose/compose-file/04-version-and-name/#version-top-level-element-obsolete) for reference.

**Comment by DevilaN** [comment]
Did docker v24 that warns about it got widely spread? If not, then people with older versions of docker would have a problem.
Warning is a bit inconvenient, but error from not having `version:` in file is no-go.

**Reply by derek-palmer** [reply]
Compose V1 is no longer supported or maintained. And hasn't received security updated in 3 years. 

Compose V2 has been in use/active development for 4 years. V2 supports almost all Compose V1 flags and subcommands as well,  it can be used as a drop-in replacement in scripts. I have yet to see this in my personal and professional docker experience of this being an issue.

[What does this mean for my projects that use Compose V1?](https://docs.docker.com/compose/migrate/#what-does-this-mean-for-my-projects-that-use-compose-v1)

[Version top-level element (obsolete)](https://github.com/compose-spec/compose-spec/blob/master/spec.md#version-and-name-top-level-elements)

https://github.com/docker/compose/issues/11628#issuecomment-2075664290

https://docs.docker.com/compose/intro/history/#introduction

![image](https://github.com/AbdBarho/stable-diffusion-webui-docker/assets/7724604/aedcda7f-0558-4217-9421-4f9d6bcc4a4f)
